mod_sample/forms.py

```diff
@@ -1,14 +1,24 @@
 """Maintains forms related to sample CRUD operations."""
 
 from flask_wtf import FlaskForm
-from wtforms import SubmitField
+from wtforms import SelectField, SubmitField, TextAreaField
+from wtforms.validators import DataRequired, ValidationError
 
-from mod_upload.forms import CommonSampleForm
+from mod_upload.models import Platform
 
 
-class EditSampleForm(CommonSampleForm):
+class EditSampleForm(FlaskForm):
     """Form to edit sample."""
 
+    notes = TextAreaField('Notes', [DataRequired(message='Notes are not filled in')])
```

Comment:
This is duplication of the fields that are in CommonSampleForm.

I'd rather see the offending fields in that class moved to the appropriate class (take a look at the history and you'll see when they've been added and for what reason).